code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -120,3 +120,11 @@
 	volume = 60
 	amount_per_transfer_from_this = 30
 	list_reagents = list("atropine" = 10, "epinephrine" = 10, "salbutamol" = 20, "spaceacillin" = 20)
+
+/obj/item/weapon/reagent_containers/hypospray/medipen/survival
+	name = "Survival Medipen"

```

Comment:
Not sure if they need this item. If they want portable quick healing they can ask chemistry